docs/azure_arc_jumpstart/azure_arc_servers/day2/arc_proxy/_index.md

```diff
@@ -0,0 +1,132 @@
+---
+type: docs
+title: "Connectivity behind a proxy server"
+linkTitle: "Connectivity behind a proxy server"
+weight: 12
+description: >
+---
+
+## Azure Arc-enabled server's connectivity behind a proxy perver
+
+The following Jumpstart scenario will guide you on how to configure your [Azure Arc-enabled server if the machine uses a proxy server](https://docs.microsoft.com/en-us/azure/cloud-adoption-framework/scenarios/hybrid/arc-enabled-servers/eslz-arc-servers-connectivity) to communicate over the internet.
+
+In this guide, you will emulate a full proxy-client configuration. The scenario will deploy both the proxy server and a client that will be Arc-enabled automatically with the agent configured to use the proxy. The automation for the proxy server deployment makes sure that the Azure Arc network service tags and IP addresses range are not blocked. To complete this process you deploy a single ARM template that will:
+
+- Deploy two Azure Linux VMs:
+  - Proxy Server.
+  - Client VM.
+- Two network security groups:
+  - Proxy Server NSG: allows inbound connections to port 22 and 3128 (proxy's port).
+  - Client VM NSG: denies all outbound connections and only allows inbound for SSH.
+- Each VM will have a custom script extension deployed:
+  - Proxy Server: will have a script to deploy and configure [Squid proxy](https://ubuntu.com/server/docs/proxy-servers-squid).
+  - Client VM: will use the custom script extension to install and configure the Connected Machine agent using the proxy connection.
+
+    > **NOTE: It is not expected for an Azure VM to be projected as an Azure Arc-enabled server. The below scenario is unsupported and should ONLY be used for demo and testing purposes.**
+
+## Prerequisites
+
+- Clone the Azure Arc Jumpstart repository
+
+    ```shell
+    git clone https://github.com/microsoft/azure_arc.git
+    ```
+
+- [Install or update Azure CLI](https://docs.microsoft.com/cli/azure/install-azure-cli?view=azure-cli-latest). Azure CLI should be running version 2.25.0** or later. Use ```az --version``` to check your current installed version.
+
+- Azure Arc-enabled servers depends on the following Azure resource providers in your subscription in order to use this service. Registration is an asynchronous process, and registration may take approximately 10 minutes.
+
+  - Microsoft.HybridCompute
+  - Microsoft.GuestConfiguration
+
+      ```shell
+      az provider register --namespace 'Microsoft.HybridCompute'
+      az provider register --namespace 'Microsoft.GuestConfiguration'
+      ```
+
+      You can monitor the registration process with the following commands:
+
+      ```shell
+      az provider show --namespace 'Microsoft.HybridCompute'
+      az provider show --namespace 'Microsoft.GuestConfiguration'
+      ```
```

Comment:
any opportunity to include this in the automation?

Reply:
I kept it as we do in any ARM based automation. If we wanted to include it I would have to add another layer as a script, which I think might be more confusing